app/codes/page.tsx

```diff
@@ -3,19 +3,22 @@
 import Image from "next/image";
 
 import { getCodesCmsData } from "@/lib/server/cms-data";
-import { Button } from "@/components/ui/button";
-import CopyButton from "@/components/CopyButton";
+import { Code } from "@/lib/cms-types";
```

Comment:
You can import this as just the type since that’s all you need from it

Reply:
Like this? `import type { Code } from "@/lib/cms-types";` Done.